source/adminguide/service_offerings.rst

```diff
@@ -169,9 +162,13 @@ To create a new compute offering:
       system VM is running. Shared allocates from storage accessible via
       NFS.
 
-   -  **Custom**: Custom compute offerings can be used in following
+   -  **Provisioning type**: The type of disk that should be allocated. 
+      Local
+
+
+   -  **Compute Offering Type**: Custom compute offerings can be used in following
       cases: deploying a VM, changing the compute offering of a stopped
-      VM and running VMs, which is nothing but scaling up.
+      VM and running VMs.
 
       If the Custom field is checked, the end-user must fill in the
```

Comment:
Can this be rephrased as there is no Custom checkbox field now

Reply:
+1

There is a dropdown now to indicate the different types of offerings (something we changed towards the finish of feature)